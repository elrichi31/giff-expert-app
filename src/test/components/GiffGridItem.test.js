const { shallow } = require("enzyme")
const { GiffGridItem } = require("../../components/GiffGridItem")
import React from 'react'

describe('Pruebas con snapshot', () => {
    const id = '1234'
    const url = 'https://localhost:6000/welcome'
    const wrapper = shallow(<GiffGridItem id={id} url={url}/>)
    const className = 'cards'
    test('debe mostrar el componente correctamente ', () => {
        expect(wrapper).toMatchSnapshot();
    })
    test('debe de mostrar el url de la imagen', () => {
        const img = wrapper.find('img');
        expect(img.prop('src')).toBe(url)
        expect(img.prop('alt')).toBe(id)
    })
    test('debe tener las clases de css', () => {
        const div = wrapper.find('img')
        expect(div.prop('className')).toBe(className)
    })
    
        
} )