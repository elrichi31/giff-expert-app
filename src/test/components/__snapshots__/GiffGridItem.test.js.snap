// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con snapshot debe mostrar el componente correctamente  1`] = `
<div>
  <img
    alt="1234"
    className="cards"
    src="https://localhost:6000/welcome"
  />
</div>
`;
